runtime/bindings/python/src/pyopenvino/graph/preprocess/pre_post_process.cpp

```diff
@@ -315,150 +305,72 @@ static void regclass_graph_OutputTensorInfo(py::module m) {
                 tensor : OutputTensorInfo
                     Reference to itself to allow chaining of calls in client's code in a builder-like manner.
               )");
-    info.def("set_layout", [](const std::shared_ptr<ov::preprocess::OutputTensorInfo>& me, const ov::Layout& layout) {
-        me->set_layout(layout);
-        return me;
+    info.def("set_layout", [](ov::preprocess::OutputTensorInfo& me, const ov::Layout& layout) {
+        return &me.set_layout(layout);
     });
 }
 
 static void regclass_graph_InputInfo(py::module m) {
-    py::class_<ov::preprocess::InputInfo, std::shared_ptr<ov::preprocess::InputInfo>> inp(m, "InputInfo");
+    py::class_<ov::preprocess::InputInfo, ref_wrapper<ov::preprocess::InputInfo>> inp(m, "InputInfo");
     inp.doc() = "openvino.impl.preprocess.InputInfo wraps ov::preprocess::InputInfo";
 
     inp.def(py::init<>(), R"(Default constructor, can be used only for networks with exactly one input)");
     inp.def(py::init<size_t>(), R"(Constructor with parameter index as argument)");
     inp.def(py::init<const std::string&>(), R"(Constructor with input tensor name as argument)");
```

Comment:
Please remove this 'init' as well - user doesn't need to create such objects (there is no place where it can be passed)

Reply:
Removed.